content/STRUCTURE.de.json

```diff
@@ -1,9 +1,14 @@
 [
     {"path": "/",
-     "title": "",
-     "menu": ["Home"],
+     "title": "Home - Home Page",
```

Comment:
I think the `title` should be left to an empty value, otherwise the page title will become `Linux Containers - Home - Home Page` which would be pretty weird.